src/panels/energy/ha-panel-energy.ts

```diff
@@ -60,31 +60,32 @@ class PanelEnergy extends LitElement {
 
   protected render(): TemplateResult {
     return html`
-      <ha-top-app-bar-fixed>
-        <ha-menu-button
-          slot="navigationIcon"
-          .hass=${this.hass}
-          .narrow=${this.narrow}
-        ></ha-menu-button>
-        <div slot="title">${this.hass.localize("panel.energy")}</div>
-        ${this.narrow
-          ? ""
-          : html`
-              <hui-energy-period-selector
-                slot="actionItems"
+      <div class="header">
+        <div class="toolbar">
+          ${!this.narrow
+            ? html`<div class="main-title">
+                ${this.hass.localize("panel.energy")}
+              </div>`
+            : html`<ha-menu-button
```

Comment:
This should also be rendered on non narrow, as there is an option to `Always hide the sidebar` in the profile.